tests-extension/test/ingress_operator.go

```
@@ -0,0 +1,1078 @@
+	g.It("Author:shudili-ROSA-OSD_CCS-ARO-Critical-62530-openshift ingress operator is failing to update router-certs [Serial]", func() {
+		var (
+			icName        = "ocp62530"
+			tlsSecretName = "custom-cert-62530"
+			dirname       = "/tmp/OCP-62530-ca/"
+		)
+
+		defer deleteIC(ctx, dynamicClient, icName)
+		defer deleteSecret(ctx, kubeClient, ingressNamespace, tlsSecretName)
+		defer func() {
+			if rmErr := os.RemoveAll(dirname); rmErr != nil {
+				fmt.Fprintf(g.GinkgoWriter, "failed to remove %s: %v\n", dirname, rmErr)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)